### ISSUE TITLE
Fix timestamp timezone

### ISSUE THREAD

**Issue by jeffreybakker** [kickin-media/media-back-end#20]
Currently, the back-end assumes that - for all timestamps - that they are in the current local time. However, this brings some issues with summer and winter time and the compatibility with the front-end. Therefore we should update timestamps such that they do not change whenever the seasons change.

My suggestion would be to save all timestamps as UTC.

**Comment by jonathanjuursema** [comment]
@jeffreybakker and I spent some time discussing how to best approach this, and what works best for the app.

As it turns out, there are several EXIF fields in the spec we can use:
- `datetime` - The time at which the actual image (in our case likely the processed JPG) was taken. This is (supposedly) in local time.
- `datetime_original` - The time at which the original image was taken. This is the time we're interested in. This is (supposedly) in local time.
- `datetime_digitized` - We think this is an alias for the original datetime?
- `offset_time` - This field is available for all three fields above (adding `_original` and `_digitized` for the lower two). This specifies the timezone of the datetime (which is in local time).

So from correcting the `datetime` by the `offset_time` we can find UTC. 

However, not all camera's have the same fields (we inspected some sample photos), so it's not very predictable what fields we have to use without assembling a list of our known camera's and maintaining that, or making assumptions.

So we went back to the question: what do we want to achieve?

We would like to make sure that the website always displays the local time, and we want to make sure that photos made by different people in the same album can be ordered chronologically.

We currently take the `datetime_original` from the EXIF data and save that as "the datetime a photo was taken" in the database. If we make clear that this time is _local time_ and don't alter it any further, we already achieve all of these goals. This solution assumes two things:

- All photographers at an event have at least the time of their camera set correctly. (Even if the timezone itself is incorrect, our solution still shows the correct local time as long as the time displayed in the camera is the same for all photographers.)
- We don't support an overview (either an album or a photostream) where we combine photos taken from different timezones. In this situation, the sorting no longer works as intended.

Finally, since we're saving the raw EXIF data for photos anyway, we can always change the behaviour down the road. This will require recalculating the desired datetime field of a photo for all photos based on the original EXIF, but is very doable. Since, at this moment, we have no use for supporting multiple timezones during the same event, we agreed not to change the behaviour of the API.

To summarize: the API will (and already does) extract local time from the photo, and sets that as _the_ datetime for a photo. It uses the localtime for sorting and filtering, and will return the unaltered _local time_ back on requests.
